src/grafana/chart/templates/uds-package.yaml

```diff
@@ -9,6 +9,9 @@ spec:
       clientId: uds-core-admin-grafana
       redirectUris:
         - "https://grafana.admin.{{ .Values.domain }}/login/generic_oauth"
+      groups:
+        anyOf:
+          - /UDS Core/Admin
```

Comment:
Do we want to include this as well? If so we should make sure Auditor is allowed.

Reply:
ahh yeah i didnt push auditor, i know we discussed this but maybe we didnt come to a decision on if we want to put this in the baseline or not.